arm/Microsoft.Network/firewallPolicies/deploy.bicep

```diff
@@ -113,7 +110,6 @@ resource firewallPolicy 'Microsoft.Network/firewallPolicies@2021-03-01' = {
     } : null
     dnsSettings: enableProxy ? {
       enableProxy: enableProxy
-      requireProxyForNetworkRules: requireProxyForNetworkRules
```

Comment:
not supported anymore?

Reply:
No. Removed from all API versions (still in the documentation, but fails with error message stating that it is not longer usable)